style.css

```diff
@@ -0,0 +1,85 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,400;0,700;1,400;1,600&display=swap");
+
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+html,
+body {
+  background-color: #fff;
+  color: #000;
+  font-family: "Poppins", sans-serif;
+}
+
+a {
+  text-decoration: none;
+  color: black;
+  cursor: pointer;
+}
+
+ul {
+  list-style: none;
+}
+
+@media (max-width: 375px) {
+  .navbar {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    color: #6070ff;
+    margin: 10px;
```

Comment:
- [ ] It is better to remove the media query for this project milestone, which you will use when you learn about responsiveness in the desktop version. Here you design a mobile-first approach with no need for media query using only margin, padding, width, and height properties to align this project milestone.